scripts/_autoloads/critters.gd

```diff
@@ -0,0 +1,104 @@
+extends Node
```

Comment:
no class name

Reply:
I believe there is a conflict with giving autoloads a class name. Also it is superfluous as an autoload will have a globally accessible unique name.

Reply:
Moved this to Brainiac. Critters is no longer an autoload, and will only load if we are in a devbuild.